pkg/apis/kops/networking.go

```diff
@@ -76,6 +76,18 @@ type CanalNetworkingSpec struct {
 	// for traffic between pod to host after calico rules have been processed.
 	// Default: ACCEPT (other options: DROP, RETURN)
 	DefaultEndpointToHostAction string `json:"defaultEndpointToHostAction,omitempty"`
+	// Controls whether Felix inserts rules to the top of iptables chains, or
```

Comment:
can we add add the names to the exported vars

Reply:
Resolved this.